clang/test/CodeGenSYCL/sycl-multi-kernel-attr.cpp

```diff
@@ -0,0 +1,21 @@
+// RUN: %clang_cc1 -std=c++11 -triple spir64-unknown-linux-sycldevice -disable-llvm-passes -fsycl-is-device -emit-llvm -o - %s | FileCheck %s
+
+class Functor {
+public:
+  [[cl::intel_reqd_sub_group_size(4), cl::reqd_work_group_size(32, 16, 16)]] void operator()() {}
```

Comment:
Can these attribute conflict with each other?
If yes, we should make sure that the diagnostic is correct in this case.

Reply:
I don't think they do.

Reply:
But these checks will be added in https://github.com/intel/llvm/pull/883 for another pair of attributes.